src/software/simulation/physics/physics_ball.h

```diff
@@ -122,12 +125,11 @@ class PhysicsBall
     // where the ball initially became in flight
     std::optional<Point> in_flight_origin;
     double in_flight_distance_meters;
+    // friction with other objects, such as robots/wall
+    static constexpr double BALL_FRICTION = 0.0;
 
-    // These restitution and friction values are somewhat arbitrary. Because this is an
-    // "ideal" simulation, we can approximate the ball as having perfectly elastic
-    // collisions and no friction. Because we also do not generally depend on specific
-    // behaviour when the ball collides with something, getting these values to perfectly
-    // match reality isn't too important.
-    static constexpr double BALL_RESTITUTION = 1.0;
-    static constexpr double BALL_FRICTION    = 0.0;
+    // The restitution is the amount of energy retained when bouncing off walls and linear
+    // damping is friction with the ground
```

Comment:
Could we add units to the variable names or add a comment if they're unitless?

Reply:
Would have to dig into Box2D to figure out the units for linear damping...

Reply:
They are some weird unitless number. 

I think restitution is between 0-1 for perfectly inelastic - perfectly elastic collisions (I'm assuming greater than 1 adds energy?)

Link to this for damping: https://gamedev.stackexchange.com/questions/160047/what-does-lineardamping-mean-in-box2d. It gives closed form models for velocity / position vs time, and the source code for calculating velocity with damping.

Reply:
I think it's fine to just say it's a unitless quantity, and indicate what the extreme values mean, if anything.

Reply:
added comment